exist-core/src/main/java/org/exist/xquery/functions/fn/FunMatches.java

```diff
@@ -46,92 +44,85 @@
 import java.util.regex.Pattern;
 import java.util.regex.PatternSyntaxException;
 
+import static org.exist.xquery.FunctionDSL.*;
+import static org.exist.xquery.functions.fn.FnModule.functionSignatures;
 import static org.exist.xquery.regex.RegexUtil.*;
 
 /**
  * Implements the fn:matches() function.
- * 
+ * <p>
```

Comment:
This new paragraph is never closed. Is that a problem?

Reply:
@line-o I think in Javadoc that is considered ok (nope... I don't know why either?!?)

Reply:
Well it is not necessary in HTML to close some elements (p, li, th, td, ...). That is why I was asking, as it could be OK.